WindowsDevicePortalWrapper/UnitTestProject/Core/EtwTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿//----------------------------------------------------------------------------------------------
+// <copyright file="EtwTests.cs" company="Microsoft Corporation">
+//     Licensed under the MIT License. See LICENSE.TXT in the project root license information.
+// </copyright>
+//----------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Microsoft.Tools.WindowsDevicePortal.DevicePortal;
+
+namespace Microsoft.Tools.WindowsDevicePortal.Tests.Core
+{
+    /// <summary>
+    /// Test class for ETW APIs.
+    /// </summary>
+    [TestClass]
+    public class EtwTests : BaseTests
+    {
+        /// <summary>
+        /// Basic test of GET method for getting a list of custom registered ETW providers.
+        /// </summary>
+        [TestMethod]
+        public void GetCustomEtwProvidersTest()
+        {
+            TestHelpers.MockHttpResponder.AddMockResponse(DevicePortal.CustomEtwProvidersApi, HttpMethods.Get);
+
+            Task<EtwProviders> getCustomEtwProvidersTask = TestHelpers.Portal.GetCustomEtwProvidersAsync();
+            getCustomEtwProvidersTask.Wait();
+
+            Assert.AreEqual(TaskStatus.RanToCompletion, getCustomEtwProvidersTask.Status);
+
+            ValidateEtwProviders(getCustomEtwProvidersTask.Result);
+        }
+
+        /// <summary>
+        /// Basic test of GET method for getting a list of registered ETW providers.
+        /// </summary>
+        [TestMethod]
+        public void GetEtwProvidersTest()
+        {
+            TestHelpers.MockHttpResponder.AddMockResponse(DevicePortal.EtwProvidersApi, HttpMethods.Get);
+
+            Task<EtwProviders> getEtwProvidersTask = TestHelpers.Portal.GetEtwProvidersAsync();
```

Comment:
I believe that the VS test infrastructure supports making methods 'public async Task' so you can 'await' on the task to make things a little cleaner and more representative; i.e. 
EtwProviders etwProviders = await TestHelpers.Portal.GetEtwProvidersAsync();